dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -21,12 +22,36 @@
 		sram0: memory@20000200 {
 			compatible = "mmio-sram";
 		};
+
 		pinctrl: pin-controller {
 			compatible = "renesas,smartbond-pinctrl";
 			status = "okay";
 			label = "PINCTRL";
 		};
 
+		gpio0: gpio@50020a00 {
+			compatible = "renesas,smartbond-gpio";
+			gpio-controller;
+			#gpio-cells = <2>;
+			reg = <0x50020a00 20
+			       0x50020a18 128
+			       0x50000070 12
+			       0x50000114 36>;
+			interrupts = <38 0>;
+			port = <0>;
+		};
+
+		gpio1: gpio@50020a04 {
```

Comment:
seems like pinctrl is re-using gpio regs, so you should probably do like STM32/GD32:

```
pinctrl: pin-controller@... {
     ...
     gpio0: gpio@... {
     };
};
```